src/github.js

```diff
@@ -156,12 +161,25 @@ const listRepoContents = async (token, repo, path = '') => {
  * @returns {string[]} - List of repos. Example: ["silinternational/vulnerability-scanner"]
  */
 module.exports.listRepos = async (org, token) => {
-  const response = await request("GET /orgs/:org/repos", {
-    headers: {
-      authorization: "token " + token,
-    },
-    org: org,
-  })
-  const repos = response.data
-  return repos.map(repo => repo.full_name)
+  const allRepos = []
+  const pageSize = 100
+  
+  let page = 0
+  let thereAreMore = true
+  while (thereAreMore) {
+    page++
+    const response = await request("GET /orgs/:org/repos?per_page=:pageSize&page=:page", {
+      headers: {
+        authorization: "token " + token,
+      },
+      org: org,
+      page: page,
+      pageSize: pageSize
```

Comment:
trailing commas?

Reply:
Oh, are they allowed in JS now? I'll have to remember that.

Reply:
should be...